controllers/openstackcontrolplane_controller.go

```diff
@@ -427,6 +452,7 @@ func (r *OpenStackControlPlaneReconciler) SetupWithManager(mgr ctrl.Manager) err
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&ospdirectorv1beta1.OpenStackControlPlane{}).
 		Owns(&corev1.Secret{}).
+		Owns(&corev1.ConfigMap{}).
```

Comment:
If we do use `CM`, I think we need to add the kubebuilder annotation for the permissions.  Also, should we use a filter function here so that not every config map in the namespace wakes the controller on a change?